src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovySandbox.java

```diff
@@ -67,6 +68,7 @@
      *
      * See {@link #createSecureCompilerConfiguration()} for the discussion.
      */
+    @SuppressFBWarnings(value = "DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED", justification = "Should be managed by the caller.")
```

Comment:
Fine, Jenkins creates `ClassLoader`s all the time, it just assumes it is running in a privileged mode.